network.py

```diff
@@ -889,11 +867,18 @@ def request_thread(result_queue):
                 })
         return return_object
 
+    def make_requests_session(self):
+        s = requests.Session()
+        s.mount(u'http://', HTTPAdapter(max_retries=REQUESTS_RETRY))
+        s.mount(u'https://', HTTPAdapter(max_retries=REQUESTS_RETRY))
+        s._reuse_count = itertools.count()
```

Comment:
Yeah, I was using it before these commits were pushed.  I'm probably going to implement session garbage collection and may use reuse_count as a mechanism for deciding when to close out old sessions.  But if not I'll yank before next PR.  Thanks!